src/shared/styles/search-tables.less

```diff
@@ -8,10 +8,6 @@
 @select-blue: rgba(217, 230, 248, 1);
 @select-black: rgba(51, 51, 51, 0.8);
 
-.height-test-wrapper {
-  height: 'calc(100vh - 82px)';
```

Comment:
Ah, it's the quotes that's breaking it...should be:

`height: calc(100vh - 82px);`

Reply:
Makes sense. Will update.